packages/@react-spectrum/style-macro-s1/src/style-macro.ts

```diff
@@ -367,7 +370,7 @@ interface Rule {
 // This maps to an alphabet containing lower case letters, upper case letters, and numbers.
 // For numbers larger than 62, an underscore is prepended.
 // This encoding allows easy parsing to enable runtime merging by property.
-function generateName(index: number, atStart = false): string {
```

Comment:
`atStart` no longer needed since it was being used to avoid class names starting with numbers, which can't happen anymore due to prefix.